### ISSUE TITLE
Bug for small BTC size order with BTCPAY

### ISSUE THREAD

**Issue by davestaxcp** [CounterpartyXCP/counterparty-lib#1278]
Looking to see if this bug mentioned here https://github.com/CounterpartyXCP/cips/discussions/96#discussioncomment-625452 can get looked at and fixed to help new users wanting to use small amounts of BTC to be able to succesfully use the BTC/TOKEN (BTCpay) function on the XCP DEx.

**Comment by adamkrellenstein** [comment]
yeah it's a good idea. the price of BTC has obv. increased massively since this magic number was set. but it's a protocol change and we may want to kill BTCPay once we have atomic swaps.

**Comment by davestaxcp** [comment]
and what about the users in this transitionary period?

while the community waits for the 

- development, 
- testing and 
- implementation of this new feature..... 

they are having to rely on BTCpay at this time....

---

what happens when a user sees this tx go through: https://xchain.io/tx/88afd2e9618afbc875c8a3743be4cf4e569a9358b6bd1b7a80279d0ccd851e4f

and gets influenced to put up their art (and follow the 'leaving the wallet open process' in the correct manner)

but their small BTC/TOKEN order does not go through because it simply is "below 0.001 BTC" (and they will have NO idea why, and much of the community will blame them for not "leaving the wallet open")....... (ive seen this happen a good deal of times)

the user will immediately think "wow this is broken"

--- 

I get its a protocol change..... but why throw away an entire functionality because "it's a little broken"  .... and then expect the user base the forcibly transition to the new atomic swap system? 

... while also giving them another reason to get frustrated and angry before waiting for the new features to go through implementation, testing and included in a wallet they trust

giving users some time to use features at their disposal while also making the feature work as intended will help the transitory process for them to see the use case of both new and old protocol functionality

---

just my two sats.... coming from the one who finally tested and wrote up this issue that has been plaguing the community for years.... 

- assume 90% of users running into the issue wont say anything about it 

- and then 99% of that remaining 10% didnt even consider logging the issue (if they even know where to find where to log it)

**Comment by adamkrellenstein** [comment]
Hm, yeah, okay. It's maybe worth fixing now. Will put it on the docket...

**Comment by ouziel-slama** [comment]
fixed with `fix_min_btc_quantity` protocol change.
